easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'mstore'
+version = '0.2.0'
+
+homepage = 'https://github.com/grimme-lab/mstore'
+description = """Molecular structure store for testing"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```

Comment:
surely this should be at `GCC` instead of `GCCcore` if we have an `intel-compilers` variant?

Reply:
Yes, that has been merged in PR #19283
I will remove that file for that reason.